buildtools/check-example

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+import tempfile
+
+
+def main():
+    current = sys.argv[1]
+    ref = sys.argv[2]
+    mask = ref.replace("-ref", "-mask")
+    diff = current.replace(".html.png", "-diff.png")
+
+    if os.path.exists(mask):
+        print("Apply mask")
+        old_current = current
+        old_ref = ref
+        current = tempfile.NamedTemporaryFile(suffix="-current.png").name
+        ref = tempfile.NamedTemporaryFile(suffix="-ref.png").name
+        subprocess.check_call(
+            ["gm", "composite", "-compose", "Multiply", old_current, main, current]
```

Comment:
Shouldn't it be "mask" instead of "main" here ?
Same on next line.

Reply:
+1 Thanks :-)